test/fixtures/unicorn/filename-case.pass.ts

```diff
@@ -0,0 +1,3 @@
+/**
+ * Find a better way to test this one.
```

Comment:
Is it a TODO? ;)

Reply:
fixed with: https://github.com/ridedott/eslint-config/pull/319/commits/7f5d5a019937421c05b398aeca35cd4901d9db26